.eslintignore

```diff
@@ -0,0 +1,4 @@
+node_modules/
```

Comment:
Do we need to have a new file? Can't we put this into `.eslintrc.json`?

Reply:
Also, do we have to duplicate here stuff from `.gitignore`?

Reply:
Hm... I can only see this: https://eslint.org/docs/user-guide/configuring#using-eslintignore-in-packagejson. So in `package.json`. I'll check how `excludedFiles` works from `eslintrc.js`.

Reply:
BTW, in https://github.com/ckeditor/ckeditor5-engine/compare/t/ckeditor5/473 you only added stuff additional to `.gitignore`.

Reply:
Yep. `node_modules/` are ignored by default.

Reply:
OK so we either stick with `.eslintignore` (with additional folders to `node_modules`) or we can add this to `package.json`:
```json
{
  "eslintIgnore": [ "src/lib/**" ]
}
```

Reply:
I'm for putting this into `package.json`. There are already other paths for linting defined there (in tasks' definitions).